content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -0,0 +1,360 @@
+│  1
+│
+◇  What is the base token price denominator?
+│  1
+│
+◇  Set this chain as default?
+│  Yes
+```
+
+The base token nominator and denominator are used to define the relation of the base token price with ETH.
+For example, if we set the nominator to 20 and the denominator to 1, together the relation is 20/1,
+this would mean that 20 tokens would be given the equivalent value as 1 ETH for gas.
+For testing purposes, we'll just use a 1:1 ratio.
+
+Next, initialize the chain in the ecosystem with:
+
+```bash
+zk_inception chain init
+```
+
+This time select no for the option to deploy a paymaster contract.
+Use the default answers for all other prompts.
+
+Now that the chain is initialized, you can start the chain server:
+
+```bash
+zk_inception server
+```
+
+## Bridging the base token to your chain
+
+The last step is to bridge some ERC20 tokens from the L1 to the new chain.
+Base tokens can not be minted on the L2 without being backed by the corresponding L1 amount. In the future, there will be more flexibility for this.
+
+Open a new terminal and use the `zksync-cli` to bridge the tokens with the command below:
+
+```bash
+npx zksync-cli bridge deposit --token 0x_YOUR_TOKEN_ADDRESS --rpc=http://localhost:3050 --l1-rpc=http://localhost:8545
+```
+
+```shell
+? Amount to deposit 5
+? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 
+? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+```
+
+To verify that this worked, let's check the new balance of our address on the L2 chain:
+
+```bash
+zksync-cli wallet balance --address 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 --rpc http://localhost:3050
+```
+
+Note that we aren't checking for any specific token address here.
+The CLI will display the amount as ETH, but in reality, it's our bridged ERC20 tokens.
+
+And that's it! You are now running a local ZK chain that uses a custom ERC20 base token.
+You can follow the steps in the previous section to deploy a contract to the new chain.
+
+## Bridging ETH to your chain
+
+If your ERC20 token bridged to the L2 "looks" like ETH, then what happens if you bridge ETH to your custom chain?
+
+The answer is that if you try to bridge regular ETH to this chain, it will just use a different token contract address.
+
+You can try this out by depositing ETH from the L1 to this new chain:
+
+```bash
+npx zksync-cli bridge deposit --rpc=http://localhost:3050 --l1-rpc=http://localhost:8545
+```
+
+```shell
+? Amount to deposit 20
+? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 
+? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+```
+
+If you run the previous command to check the base token balance again, note how the balance is still `5`.
+This is because you have to look at the L2 token balance for ETH.
+To find the L2 token address for ETH, you can use the `l2TokenAddress` method available through `zksync-ethers`.
```

Comment:
```suggestion
If you run the previous command to check the base token balance again, note how the balance is still `5`. Because ETH is no longer the base token, it has a different token address on L2 you will need to reference.
To find the L2 token address for ETH, you can use the `l2TokenAddress` method available through `zksync-ethers`.
```